apps/docs/src/guide/fuels-cli/commands.md

```diff
@@ -180,3 +180,31 @@ npx fuels@{{fuels}} versions
 
 You have all the right versions! ⚡
 ```
+
+## `fuels info`
+
+Outputs helpful information that can assist with possible misconfigurations and general information you can paste when creating new issues.
+
+```console-vue
+npx fuels@{{fuels}} info
+```
+
+```console-vue
+{
+  "sdkVersions": {
+    "FORC": "0.62.0",
+    "FUEL_CORE": "0.31.0",
+    "FUELS": "0.92.1"
+  },
+  "toolchainVersions": {
+    "error": null,
+    "systemForcVersion": "0.61.1",
+    "systemForcPath": "forc",
+    "systemFuelCoreVersion": "0.30.0",
+    "systemFuelCorePath": "fuel-core"
+  },
+  "nodeVersion": "v22.3.0",
+  "osType": "Darwin",
+  "osArch": "arm64"
+}
+```
```

Comment:
I assume this was mean't to go in:
- https://github.com/FuelLabs/fuels-ts/pull/2802

Reply:
Accidental copy from [here](https://github.com/FuelLabs/fuels-ts/pull/2738#discussion_r1685566897) I believe.